client/resource_group/controller/controller.go

```diff
@@ -468,7 +483,7 @@ func newGroupCostController(
 		gc.handleRespFunc = gc.handleRawResourceTokenResponse
 	}
 
-	gc.mu.consumption = &rmpb.Consumption{}
+	gc.mu.consumption = &rmpb.Consumption{SqlLayerCpuTimeMs: getSQLProcessCPUTime(mainCfg.isSingleGroupByKeyspace)}
```

Comment:
I think should make the `lastRequestConsumption` init with it. then `collectRequestAndConsumption` will only calculate the delta part.

Reply:
You are right, the run.consumption represents the total, not the delta